_staff/test_5.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: staff
+title: test5
+image: https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/44.png
+desc: |
+  description for person 1
+---
```

Comment:
Are we moving staff descriptions from individual files to one big YAML? I don't remember, but I would be in support of such a move.

Reply:
Yes we are! I'll work on it.